src/main/java/org/datadog/jmxfetch/Instance.java

```diff
@@ -485,6 +526,13 @@ public List<Metric> getMetrics() throws IOException {
                 }
             }
         }
+        jmxBean.setBeanCount(beans.size());
+        jmxBean.setAttributeCount(matchingAttributes.size());
+        jmxBean.setMetricCount(metrics.size());
+        log.debug("Updated jmx bean for instance: " + this.getCheckName() + " for bean: "
+                + jmxBean.toString() + " With number of beans = " + jmxBean.getBeanCount()
```

Comment:
what does `jmxBean.toString()` look like here? Do you have an example of this log line?